library/objective-c/EnvoyEngine.m

```diff
@@ -237,19 +234,17 @@ - (instancetype)initWithHandle:(uint64_t)handle observer:(EnvoyObserver *)observ
   _platformObserver = observer;
 
   // Create callback context
-  ios_context *context = (ios_context *)malloc(sizeof(ios_context));
```

Comment:
Did these actually cause problems?

Reply:
nope, the cast is just not needed. I'll open another PR to homogenize this style.